client/AddForm.js

```diff
@@ -38,14 +38,30 @@ Vue.component('add-form', {
 		}
 	},
 	template: `
-		<div class="add-form">
-			<span>Problem:</span> <input type="text" v-model="problem">
-			<br />
-			<span>Solution:</span> <textarea rows=10 cols=40 v-model="solution"></textarea>
-			<br />
-			<span>Keywords:</span> <input type="text" v-model="keywordString"> {{ keywords }}
-			<br />
-			<input type="button" value="Submit" v-on:click="submit">
+		<div class="add-form form-horizontal">
+			<div class="form-group">
+				<label for="problem" class="control-label col-sm-2">Problem:</label>
```

Comment:
iirc BS 4 doesn't need any classes on the label

Reply:
I am leaving this to figure out in v1.0